packages/artemis/src/modules/users/statistics/value-edit.tsx

```diff
@@ -1,24 +1,26 @@
 import { Editable, EditablePreview, EditableInput } from "@chakra-ui/react";
 import { useMutation } from "@hooks/useMutation";
-import { BranchMember, User, UserMetadata } from "@prisma/client";
-import { route } from "next/dist/server/router";
+import { User, UserMetadata } from "@prisma/client";
+import { useSession } from "next-auth/react";
 import { useRouter } from "next/router";
 
 export const EditableValue: React.FC<{
   metadata: UserMetadata;
 }> = ({ metadata: md }) => {
+  const session = useSession({ required: false });
   const { query } = useRouter();
   const edit = useMutation<
     UserMetadata,
-    { key: string; memberId: string; value: number }
-  >("/members/metadata", "patch", `/members/${query.id}/metadata`);
+    { key: string; userId: string; value: number }
+  >("/users/metadata", "patch", `/users/${query.id}/metadata`);
 
   return (
     <Editable
       defaultValue={md.value.toString()}
       onSubmit={async (value) => {
         await edit({
-          memberId: query.id as string,
+          // TODO: will always fall back to the current user, implement change other's statistics
```

Comment:
Is there ever a case where this would need a fallback?

Reply:
That component will be reused as a way to edit other people's statistics, which needs to be done (it is part of branches routes - which still needs to be deleted).  Right now the user can only edit their own stats

Reply:
The user shouldnt be able to edit their own stats